### PR TITLE
test: fix ginkgo unit test rule, missing test data.

### LINKED ISSUE #18: Missing testdata/sysfs.tar.bz2 breaking some of the TA and cpuallocator unit test.
```
--- FAIL: TestPoolCreation (0.00s)
panic: open testdata/sysfs.tar.bz2: no such file or directory [recovered]
        panic: open testdata/sysfs.tar.bz2: no such file or directory

goroutine 158 [running]:
...
I0130 16:47:04.972066  178521 klogcontrol.go:181] Logging to journald, forcing headers off...
--- FAIL: TestAllocatorHelper (0.00s)
    cpuallocator_test.go:38: failed to decompress testdata: open testdata/sysfs.tar.bz2: no such file or directory
FAIL
```

### PR DESCRIPTION
**Pull request by fmuyassarov**
Currently unit tests are missing sysfs testdata. This PR copies the same testdata
as it was in the cri-resmgr repo since the unit tests are the same and based on the
same test data. 

@klihub @jukkar PTAL.

Fixes: https://github.com/jukkar/nri-resmgr/issues/18

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -198,7 +198,8 @@ ginkgo-tests:
 	    --covermode atomic \
 	    --output-dir $(COVERAGE_PATH) \
 	    --junit-report junit.xml \
-	    --coverprofile coverprofile \
+	    --coverprofile $(COVERAGE_PATH)/coverprofile \
```
Comment (klihub) [inline Makefile]: nitpick/no need to update this now, just a comment for future reference: This change is unrelated to fixing the  missing sysfs testdata. Therefore IMO, it should be a separate commit of its own, ideally committed with something like `git commit -s -m "Makefile: fix ginkgo coverprofile generation."` or something to that effect. Then if those two commits were filed as a single PR (I can see the justification for that), a more suitable title could be something like `test: fix ginkgo unit test rule, missing test data.`

Again, no need to change it now. Just trying to describe how we usually prefer to go with commits (each commit a single self-contained logical unit of change) and PRs.

**Review by klihub — APPROVED** [review]
LGTM.

File: Makefile (hunk 1 above)
Comment (fmuyassarov) [reply Makefile]: yep got it. I will decouple that commit and file a separate PR. Thanks @klihub

**Review by klihub — APPROVED** [review]
LGTM

**Comment by fmuyassarov** [comment]
last push was about updating the commit title. So, I guess it is good to go now.

File: Makefile (hunk 1 above)
Comment (fmuyassarov) [reply Makefile]: I should admit I'm really bad with separating the commits but I will follow your instructions next time.

**Review by klihub — APPROVED** [review]
LGTM.

**Review by jukkar — APPROVED** [review]

**Pull request merged**